packages/accordions/src/styled/accordion/StyledInnerPanel.ts

```diff
@@ -6,37 +6,21 @@
  */
 
 import styled from 'styled-components';
-import {
-  retrieveComponentStyles,
-  DEFAULT_THEME,
-  getLineHeight
-} from '@zendeskgarden/react-theming';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'accordions.step_inner_panel';
 
-interface IStyledInnerPanel {
-  isExpanded?: boolean;
-  isAnimated?: boolean;
-}
-
-/**
- * 1. Override the inline max-height style used for animation.
- */
-export const StyledInnerPanel = styled.div.attrs<IStyledInnerPanel>({
+export const StyledInnerPanel = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
-})<IStyledInnerPanel>`
-  transition: ${props => props.isAnimated && 'max-height 0.25s ease-in-out'};
-  /* stylelint-disable-next-line declaration-no-important */
-  max-height: ${props => !props.isExpanded && '0 !important'}; /* [1] */
-  overflow: hidden;
-  line-height: ${props => getLineHeight(props.theme.space.base * 5, props.theme.fontSizes.md)};
-  font-size: ${props => props.theme.fontSizes.md};
+})`
+  overflow: inherit;
```

Comment:
`overflow` is not an inherited property. https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#formal_definition

Reply:
I believe it is ([please see spec](https://www.w3.org/TR/css-cascade-4/#inheriting)). Also in the MDN docs that you shared, I do see `inherit` listed under `global values`.

Reply:
One of the reasons I prefer `inherit` here is that the `InnerPanel` component can be removed in the next major, and treating it as a pass through will make that migration easier in the future.

Reply:
Updated to explicitly set to `hidden` for shadow DOM use cases!